Cesium3DTilesSelection/src/TilesetContentLoaderResult.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include "TilesetContentLoader.h"
+
+#include <Cesium3DTilesSelection/ErrorList.h>
+#include <Cesium3DTilesSelection/Tile.h>
+#include <CesiumAsync/IAssetAccessor.h>
+#include <CesiumGeometry/Axis.h>
+
+#include <memory>
+#include <vector>
+
+namespace Cesium3DTilesSelection {
+struct LoaderCreditResult {
+  std::string creditText;
+
+  bool showOnScreen;
+};
+
+struct TilesetContentLoaderResult {
+  std::unique_ptr<TilesetContentLoader> pLoader;
+
+  std::unique_ptr<Tile> pRootTile;
+
+  CesiumGeometry::Axis gltfUpAxis{CesiumGeometry::Axis::Y};
```

Comment:
This is set in a number of places, but if it's used anywhere I can't find it. More generally, it just seems out of place to be here. Is it needed?